bagisto/bagisto
# Fixed Order Status Issue #5125

## Issue Reference
Link: https://github.com/bagisto/bagisto/issues/5125

## Description
- If an order is already completed and the total quantity order is not equal to the refunded quantity then it will be considered as completed.

## How To Test This?
- Place an order of 2 qty (same products).
- Generate invoice and then create shipment of both quantity and refund 1.
- Order should be completed.
